app/lib/world/world.coffee

```diff
@@ -260,6 +260,11 @@ module.exports = class World
         system.start @thangs
       catch err
         console.error "Error starting system!", system, err
+    # Silently clamp health after systems run.
+    hero = _.find(@thangs, (thang) -> thang.id == 'Hero Placeholder')
+    if hero? and level.recommendedHealth?
```

Comment:
If this looks good, I can implement `maximumHealth` here as well.
Not sure if it is worth refactoring out this into a World method yet. What do you think?